### ISSUE TITLE
Support Windows authentication in MS SQL Server

### ISSUE THREAD

**Issue by vishvam430** [airbytehq/airbyte#9557]
## Tell us about the problem you're trying to solve
What are you trying to do, and why is it hard? A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
Trying to login to SQL server through Windows Authentication since our org doesn't allow database authentication for security reasons
## Describe the solution you’d like
A clear and concise description of what you want to see happen, or the change you would like to see
Provide Windows Authentication option in MS SQL Server connector
## Describe the alternative you’ve considered or used
A clear and concise description of any alternative solutions or features you've considered or are using today.
No alternative option available
## Are you willing to submit a PR?

Remove this with your answer :-)

**Comment by marcosmarxm** [comment]
Zendesk ticket #[2174](https://airbyte7538.zendesk.com/agent/tickets/2174) has been linked to this issue.

**Comment by MrMikeFloyd** [comment]
Any roadmap on this feature, or at least a workaround? If this feature isn't provided anytime soon, would you be able to provide some guidance on how to implement this with a custom connector?

Thanks!

**Comment by marcosmarxm** [comment]
@airbytehq/dbsources can someone give more context/info about where this issue is in the roadmap as now MSSQL is a certified connector?

**Comment by evantahler** [comment]
Not currently on the roadmap - certified means we can maintain the connector, not that every feature (e.g. auth method) is implemented.  As this issue gains more upvotes, we will prioritize it in the future.

**Comment by antidiminutive** [comment]
@evantahler, what is the typical upvote level that leads you to include a new "feature" on the roadmap?

**Comment by evantahler** [comment]
> @evantahler, what is the typical upvote level that leads you to include a new "feature" on the roadmap?


It's very contextual, depending on other priorities/projects/bugs/initiative we have at the moment.  We don't publish a threshold.

**Comment by bilodeauj** [comment]
We'd also like to see this feature implemented

**Comment by cloudjesper** [comment]
Lack of this feature probably disqualifies Airbyte as a candidate for our organization

**Comment by WNCRifleman** [comment]
I have to agree with @cloudjesper.  For organizations that are heavy on MSSQL.  It makes the entire Airbyte software kind of a non-starter.  I'm not sure Airbyte should advertise that they even have a full-blown MSSQL connector if it is only a partial when it comes to authentication methods.  Having no future plans for an authentication method as widely used as Windows, seems disingenuous or at a minimum apathetic.
